src/components/container/LabResultsBloodType/LabResultsBloodType.tsx

```diff
@@ -0,0 +1,149 @@
+        getLabResultsSummary();
+
+        MyDataHelps.on("externalAccountSyncComplete", getLabResultsSummary);
+        MyDataHelps.on("applicationDidBecomeVisible", getLabResultsSummary);
+        return () => {
+            MyDataHelps.off("externalAccountSyncComplete", getLabResultsSummary);
+            MyDataHelps.off("applicationDidBecomeVisible", getLabResultsSummary);
+        }
+    }, []);
+
+    if (!model) {
+        return <div ref={props.innerRef} className="mdhui-health-preview-section"><LoadingIndicator /></div>
+    }
+
+    if (model && (!model.BloodTypeLabs || model.BloodTypeLabs.length == 0)) {
+        return null;
+    }
+
+    // support any onClick function, or default to an english/spanish link
+    function drilldown() {
+        if (props.onClick) {
+            props.onClick();
+            return;
+        }
+        if (!props.externalInfoLink) { return; }
+        if (!props.externalInfoLinkDefault) { return; }
+        var linkTarget: string;
+        linkTarget = props.externalInfoLinkDefault;
+        if (MyDataHelps.language === "es" && props.externalInfoLinkSpanish) linkTarget = props.externalInfoLinkSpanish;
+        MyDataHelps.openEmbeddedUrl(linkTarget);
+    }
+
+    // Sometimes the data will only have either the type or the rH factor
+    // Combine those into a unified value, if the first value doesn't contain both
+    var summaryResults = function(bloodTypeLabs: any) {
+        var type = "";
+        var rH = "";
+        for (const bloodTypeLab of bloodTypeLabs) {
+            if (!bloodTypeLab?.MostRecentValue) continue;
+            var mostRecentValue = bloodTypeLab.MostRecentValue.toString().toLowerCase();
+            if (type == "") {
+                if (mostRecentValue == "a" || mostRecentValue.startsWith("a ")) {
+                    type = "A";
+                } else if (mostRecentValue == "ab" || mostRecentValue.startsWith("ab ")) {
+                    type = "AB";
+                } else if (mostRecentValue == "b" || mostRecentValue.startsWith("b ")) {
+                    type = "B";
+                } else if (mostRecentValue == "o" || mostRecentValue.startsWith("o ")) {
+                    type = "O";
+                }
+            }
+            if (rH == "") {
+                if (mostRecentValue.includes("pos")) {
+                    rH = "Positive";
+                } else if (mostRecentValue.includes("neg")) {
+                    rH = "Negative";
+                }
+            }
+        }
+        return type + " " + rH;
+    }
+
+    return <div ref={props.innerRef} >
+        <Action
+        bottomBorder
+        title={ language ("blood-type") }
+        titleIcon={<img className="mdhui-health-preview-icon" src={ iconBloodType} alt="Blood Type" />}
+        onClick={ model ? drilldown: undefined }
+        indicatorIcon={ faInfoCircle }
+        indicatorPosition= "topRight" 
+        className="mdhui-lab-results-summary mdhui-health-preview-section">
```

Comment:
Would update classes and trim down the CSS to only the classes used here...which are very few...I think actually if you just use the classes from HealthPreviewSection you might be able to remove the CSS file entirely?  What happens if you remove mdhui-lab-results-summary and the CSS file and just import the CSS file from health preview section?

Reply:
You were correct - they were all un-used.  Fixed